From 2f6f6b525f52e458eb1c023b8467e8ab75f795d3 Mon Sep 17 00:00:00 2001
From: Matt Coster <matt.coster@imgtec.com>
Date: Mon, 29 Jun 2026 16:47:30 +0100
Subject: [PATCH 01480/15333] dt-bindings: gpu: img,powervr-*: Remove Matt
 Coster as maintainer

I'm leaving Imagination; remove myself as a maintainer. A separate patch
leaves a personal forwarding address in the .mailmap.

Acked-by: Matt Coster <opensource@mtcoster.net>
Acked-by: Alessio Belle <alessio.belle@imgtec.com>
Link: https://patch.msgid.link/20260629-goodbye-v1-2-3bab53a80c53@imgtec.com
Signed-off-by: Matt Coster <matt.coster@imgtec.com>
---
 Documentation/devicetree/bindings/gpu/img,powervr-rogue.yaml | 1 -
 Documentation/devicetree/bindings/gpu/img,powervr-sgx.yaml   | 1 -
 2 files changed, 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/gpu/img,powervr-rogue.yaml b/Documentation/devicetree/bindings/gpu/img,powervr-rogue.yaml
index 91e4ff61b394..a6bccb600a57 100644
--- a/Documentation/devicetree/bindings/gpu/img,powervr-rogue.yaml
+++ b/Documentation/devicetree/bindings/gpu/img,powervr-rogue.yaml
@@ -8,7 +8,6 @@ $schema: http://devicetree.org/meta-schemas/core.yaml#
 title: Imagination Technologies PowerVR and IMG Rogue GPUs
 
 maintainers:
-  - Matt Coster <matt.coster@imgtec.com>
   - Alessio Belle <alessio.belle@imgtec.com>
   - Luigi Santivetti <luigi.santivetti@imgtec.com>
 
diff --git a/Documentation/devicetree/bindings/gpu/img,powervr-sgx.yaml b/Documentation/devicetree/bindings/gpu/img,powervr-sgx.yaml
index 3e7df7344430..58b799b8e96e 100644
--- a/Documentation/devicetree/bindings/gpu/img,powervr-sgx.yaml
+++ b/Documentation/devicetree/bindings/gpu/img,powervr-sgx.yaml
@@ -9,7 +9,6 @@ $schema: http://devicetree.org/meta-schemas/core.yaml#
 title: Imagination Technologies PowerVR SGX GPUs
 
 maintainers:
-  - Matt Coster <matt.coster@imgtec.com>
   - Alessio Belle <alessio.belle@imgtec.com>
   - Luigi Santivetti <luigi.santivetti@imgtec.com>
 
-- 
2.55.0

